backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlQuery.java

```diff
@@ -460,7 +460,7 @@ private List<Condition> jsonFilterQueryConditions(
                 whereCondition(
                     c.getTableName(),
                     c.getName(),
-                    c.getColumnType(),
+                    c.getColumnType().getBaseType(),
```

Comment:
unclear to me when to use baseType en when to use type , maybe would be more OO to have object handle this internally instead of caller